portfolio/src/main/webapp/index.html

```diff
@@ -1,18 +1,80 @@
 <!DOCTYPE html>
+
+<html>
+<head>
+<style>
+div {
+  background-color: whitesmoke;
+  width: 300px;
+  padding: 50px;
+  margin: 20px;
+}
+</style>
+</head>
+<body>
+
+
+</body>
+</html>
+
 <html>
   <head>
     <meta charset="UTF-8">
-    <title>My Portfolio</title>
+    <title>Tayyaba's Portfolio</title>
     <link rel="stylesheet" href="style.css">
     <script src="script.js"></script>
   </head>
   <body>
     <div id="content">
-      <h1>My Portfolio</h1>
-      <p>This is my portfolio.</p>
-      <p>Click here to get a random message:</p>
-      <button onclick="addRandomGreeting()">Hello</button>
-      <div id="greeting-container"></div>
+      <h1>Tayyaba's Portfolio</h1>
+      <p>Hi! My name is Tayyaba and welcome to my page! I am a undergraduate student at Cornell University, Class of 2022. I am majoring in Computer Science and Asian Studies.
```

Comment:
The line length might be a bit long. The style guide doesn't seem to enforce a strict length and it seems to depend on teams. I anecdotally saw 100 characters limit, I will send you an email regarding that.


Depending on the language, we usually have a style guide. For HTML, externally, you can refer to [here](https://google.github.io/styleguide/htmlcssguide.html.